samples/Sentry.Samples.Maui/MauiProgram.cs

```diff
@@ -22,6 +24,11 @@ public static MauiApp CreateMauiApp()
 
                 options.Debug = true;
                 options.SampleRate = 1.0F;
+                options.BeforeCaptureScreenshot ( (SentryEvent @event, SentryHint hint) =>
```

Comment:
We rather avoid adding dedicated callbacks like that.
For that reason, we added a `Hint` to the API. So we can add stuff like screenshots to the hint, and users can delete the screenshot if they prefer not to capture it.

Reply:
Would you like me to remove it from the sample code then?

Reply:
I meant to say I don't think we need another BeforeSend style callback because we have already BeforeSend, and the screenshot should be on the Hint. 

That's said we might solve the user problem with the callback Roman described. Where the user returns bool whether they want to capture or not. 

A separate PR can add the screenshot to the Hint so they user can also remove it if they don't want it sent at all